passing-args-to-yield/solution.md

```diff
@@ -5,16 +5,14 @@
 For comparison, here is a possible solution, so you can compare notes:
 
 ```js
-module.exports = function *generate() {
+module.exports = function *multiplier() {
   var num = 0,
-      multiplier = 1;
+      mul = 1;
 
   while(true) {
     num++;
-    multiplier = yield num * multiplier;
-    if (!multiplier) {
-      multiplier = 1;
-    }
+    mul = yield num * mul;
+    mul = mul || 1;
```

Comment:
I'd prefer to keep the explicit `if` here, just so folks who may be new to the language don't have to think about how the logic operator works on top of figuring out generators.